packages/nimble-components/src/chip/styles.ts

```diff
@@ -0,0 +1,98 @@
+import { css } from '@ni/fast-element';
+import {
+    actionRgbPartialColor,
+    bodyDisabledFontColor,
+    bodyFont,
+    bodyFontColor,
+    borderRgbPartialColor,
+    borderWidth,
+    controlHeight,
+    iconColor,
+    iconSize,
+    mediumPadding,
+    smallPadding
+} from '../theme-provider/design-tokens';
+import { display } from '../utilities/style/display';
+import { appearanceBehavior } from '../utilities/style/appearance';
+import { ChipAppearance } from './types';
+
+export const styles = css`
+    ${display('inline-flex')}
+
+    :host {
+        height: ${controlHeight};
+        width: fit-content;
+        max-width: 300px;
+        color: ${bodyFontColor};
+        font: ${bodyFont};
+        padding: 0 ${mediumPadding};
+        gap: 4px;
+        background-color: transparent;
+        border: ${borderWidth} solid rgba(${actionRgbPartialColor}, 0.3);
+        border-radius: 4px;
+        justify-content: center;
+        align-items: center;
+        background-repeat: no-repeat;
+    }
+
+    :host([disabled]) {
+        cursor: default;
+        color: ${bodyDisabledFontColor};
+        border-color: rgba(${borderRgbPartialColor}, 0.3);
+        box-shadow: none;
+        background-image: none;
+        background-size: 100% 100%;
+    }
+
+    :host([disabled])::before {
+        box-shadow: none;
+    }
+
+    :host([disabled]) slot[name='start']::slotted(*),
+    :host([disabled]) .remove-button {
```

Comment:
Why do you need to set disabled styling on the remove button? Isn't the remove button hidden when the chip is disabled?

Reply:
Yup. Removed.